<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wppf_options_dialog</class>
 <widget class="QDialog" name="wppf_options_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>393</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="4" column="2">
    <widget class="QPushButton" name="select_experiment_file_button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Select File</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="wppf_method_label">
     <property name="text">
      <string>WPPF Method:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QComboBox" name="background_method">
     <item>
      <property name="text">
       <string>chebyshev</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>spline</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="wppf_method">
     <item>
      <property name="text">
       <string>LeBail</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Rietveld</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="experiment_file">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="background_method_label">
     <property name="text">
      <string>Background Method:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QTableWidget" name="table">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Minimum</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Maximum</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Vary</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QGridLayout" name="background_method_parameters_layout">
     <item row="1" column="0">
      <widget class="QLabel" name="chebyshev_polynomial_degree_label">
       <property name="text">
        <string>Chebyshev Polynomial Degree:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="chebyshev_polynomial_degree">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100000000</number>
       </property>
       <property name="value">
        <number>6</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="refinement_steps_label">
     <property name="text">
      <string>Refinement Steps:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QSpinBox" name="refinement_steps">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10000000</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="use_experiment_file">
     <property name="text">
      <string>Use Experiment File</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>wppf_method</tabstop>
  <tabstop>refinement_steps</tabstop>
  <tabstop>background_method</tabstop>
  <tabstop>chebyshev_polynomial_degree</tabstop>
  <tabstop>use_experiment_file</tabstop>
  <tabstop>experiment_file</tabstop>
  <tabstop>select_experiment_file_button</tabstop>
  <tabstop>table</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>wppf_options_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>wppf_options_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_experiment_file</sender>
   <signal>toggled(bool)</signal>
   <receiver>experiment_file</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_experiment_file</sender>
   <signal>toggled(bool)</signal>
   <receiver>select_experiment_file_button</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>585</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
